modules/auth/cms_authentication_module.e

```diff
@@ -125,6 +126,29 @@ feature -- Hooks configuration
 
 feature -- Handler
 
+	handle_account (api: CMS_API; req: WSF_REQUEST; res: WSF_RESPONSE)
+		local
+			r: CMS_RESPONSE
+		do
+			create {GENERIC_VIEW_CMS_RESPONSE} r.make (req, res, api)
+			r.set_value ("Account Info", "optional_content_type")
+
+			if attached template_block ("account_info", r) as l_tpl_block then
+				r.set_value (current_user (req), "user")
+				if attached current_user (req) as l_user  then
+					r.set_value (api.user_api.user_roles (l_user), "roles")
+				end
+				r.add_block (l_tpl_block, "content")
+			else
+				debug ("cms")
```

Comment:
I am wondering, if we should not raise an error message ... or maybe only if current user has a specific permission ...

"view internal error"
"view internal warning"

and so on ..